### PR TITLE
Fix Release action

### PR DESCRIPTION
**Pull request by baizhenyu**

### CODE REVIEW

File: release/action.yml [hunk 1]
```diff
@@ -6,8 +6,83 @@ outputs:
...
     description: Build number
 runs:
-  using: 'docker'
-  image: 'Dockerfile'
-  args:
-    - ${{ inputs.branch }}
-    - ${{ inputs.buildNum }}
+  using: 'composite'
+  steps:
+    - run: |
```
Comment (SheranWiratungaD2L) [inline release/action.yml:11]: FYI I think you can still have this in a separate script, and call it from the action. I'm fine with it the way it is, though.

Comment (baizhenyu) [reply release/action.yml:11]: Spent last hour trying to keep it in a separate file but getting the following error:
```
/home/jparker/actions-runner/_work/_actions/Brightspace/bdp-ci-action/release/release/entrypoint.sh not found
```
I followed instruction here: https://docs.github.com/en/actions/creating-actions/creating-a-composite-run-steps-action:
```
- run: ${{ github.action_path }}/entrypoint.sh
     shell: bash
```
Not sure what am I missing.

Comment (baizhenyu) [reply release/action.yml:11]: Just realized I might miss a `release` in the PATH. But I tried to use 'ls /home/jparker/actions-runner/_work/_actions/Brightspace/bdp-ci-action/release/release/' to debug it also throws file/dir not exist error.

Comment (SheranWiratungaD2L) [reply release/action.yml:11]: Oh, interesting. (If you want to keep investigating) Maybe try running `ls` in `${{ github.action_path }}`? I wonder if there's something weird going on with the path.

Comment (baizhenyu) [reply release/action.yml:11]: Tried that as well. Also got file/dir not exist error. I was wondering if it was some weird permission issue.

**Comment by baizhenyu** [comment]
Merging this PR for now to unblock other work. Will keep investigating the above mentioned issue.

**Pull request merged**